crates/levm/tests/tests.rs

```diff
@@ -1127,17 +1136,195 @@ fn mload_uninitialized_memory() {
         .flat_map(Operation::to_bytecode)
         .collect::<Bytes>();
 
-    let mut vm = VM::new(bytecode);
+    let mut vm = VM::new(bytecode, Address::zero(), U256::zero());
 
     vm.execute();
 
-    let memory_size = vm.current_call_frame().stack.pop().unwrap();
-    let loaded_value = vm.current_call_frame().stack.pop().unwrap();
+    let memory_size = vm.current_call_frame_mut().stack.pop().unwrap();
+    let loaded_value = vm.current_call_frame_mut().stack.pop().unwrap();
 
     assert_eq!(loaded_value, U256::zero());
     assert_eq!(memory_size, U256::from(96));
 }
 
+#[test]
+fn call_returns_if_bytecode_empty() {
+    let callee_bytecode = vec![].into();
+
+    let callee_address = Address::from_low_u64_be(U256::from(2).low_u64());
+    let callee_address_u256 = U256::from(2);
+    let callee_account = Account::new(U256::from(500000), callee_bytecode);
+
+    let caller_ops = vec![
+        Operation::Push32(U256::from(100_000)), // gas
+        Operation::Push32(callee_address_u256), // address
+        Operation::Push32(U256::zero()),        // value
+        Operation::Push32(U256::from(0)),       // args_offset
+        Operation::Push32(U256::from(0)),       // args_size
+        Operation::Push32(U256::from(0)),       // ret_offset
+        Operation::Push32(U256::from(32)),      // ret_size
+        Operation::Call,
+        Operation::Stop,
+    ];
+
+    let mut vm = new_vm_with_ops_addr_bal(
+        &caller_ops,
+        Address::from_low_u64_be(U256::from(1).low_u64()),
+        U256::zero(),
+    );
+
+    vm.add_account(callee_address, callee_account);
+    println!("to excec");
+    vm.execute();
+
+    let success = vm.current_call_frame_mut().stack.pop().unwrap();
+    assert_eq!(success, U256::one());
+}
+
+fn callee_return_bytecode(return_value: U256) -> Bytes {
```

Comment:
just a small thing, but maybe we could move this function at the top here seems like another tests and it's a little confusing

Reply:
done!